cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/SynchronousMachineConversion.java

```diff
@@ -60,6 +60,14 @@ public void convert() {
         if (p.asInt("referencePriority", 0) > 0) {
             SlackTerminal.reset(g.getTerminal().getVoltageLevel(), g.getTerminal());
         }
+        if (p.containsKey("normalPF")) {
+            // Convert to double then to string to have canonical representation of number
+            g.setProperty("GeneratingUnit.normalPF", Double.valueOf(p.get("normalPF")).toString());
+        }
+        String generatingUnit = p.getId("GeneratingUnit");
+        if (generatingUnit != null) {
+            g.setProperty("GeneratingUnit", generatingUnit);
```

Comment:
Just for me, can a generating unit be linked to several synchronous machines?

Reply:
Yes, a `GeneratingUnit` can contain one or more synchronous machines. From CGMES doc, a `GeneratingUnit` is defined as _A single or set of synchronous machines for converting mechanical power into alternating-current power_.